.github/workflows/deploy.yml

```
@@ -0,0 +1,33 @@
+name: Deploy to GitHub Pages
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
```

Please don't run deploy jobs on Pull Requests with out filter !
```suggestion
on:
  # Runs on pushes targeting the default branch
  push:
    branches: [main]
```

---

@dmohns please check again